.devcontainer/router.yaml

```diff
@@ -0,0 +1,9 @@
+headers:
+  all: # Header rules for all subgraphs
+    request:
+      - propagate:
+          matching: .*
+          
+supergraph:
+  listen: 0.0.0.0:4000
+  
```

Comment:
Whitespace weirdness going on here

Reply:
EOF and a line between entities? 🤔

Reply:
Nah, there seems to be whitespace on line 8, means that it's being treated as a "real" line